packages/backend/middlewares/auth.js

```diff
@@ -54,11 +54,33 @@ const jwtAuth = (req, res, next) => {
   }
 
   req.address = addressToken;
+  req.isAdmin = tokenContents.claims.isAdmin;
 
-  console.log("success!");
   next();
 };
 
+/**
+ * Middleware to validate logged in admin requests.
+ *
+ * @param {Express.Request} req
+ * @param {Express.Response} res
+ * @param {Express.NextFunction} next
+ */
+const jwtAdminAuth = (req, res, next) => {
+  jwtAuth(req, res, () => {
+    // Added by jwtAuth
+    if (!req.isAdmin) {
+      console.log("returning 401, Not an admin");
+      res.sendStatus(401);
+      return;
+    }
+
+    next();
+  });
```

Comment:
@dgrcode Added the logic inside the "next" function to be sure `jwtAuth` was successful. 

If not, if `jwtAuth` throws a 401 and `jwtAdminAuth` consecutively throws another 401:
`Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client`

Reply:
yep! I wasn't sure about how to chain middlewares, but using `next` looks like the best solution to me.

`jwtAuth` should only reject if the token itself is invalid. The admin rejection should happen in `jwtAdminAuth`.